src/core/server/app/middleware/graphql/apolloServer.ts

```diff
@@ -45,6 +47,9 @@ function contextProvider(options: ContextProviderOptions) {
       opts.clientID = clientID.slice(0, 36);
     }
 
+    // Set trace ID for debug events if we error
+    req.traceID = opts.traceID;
+
```

Comment:
This shouldn't be duplicated as it's already available on the request as `req.coral.id`.